src/lib.rs

```diff
@@ -9,11 +9,12 @@ pub mod io;
 #[allow(dead_code)]
 pub mod tensor;
 
-use crate::dlpack_py::cvtensor_to_dlpack;
-use crate::io::read_image_rs;
-
-#[cfg(feature = "libjpeg-turbo")]
 use crate::io::read_image_jpeg;
+use crate::io::read_image_rs;
+use crate::io::write_image_jpeg;
```

Comment:
Personally, I would like to have simpler names for these methods in kornia_rs -- like just `read_image` and `write_image` and have a flag/enum/anything to choice the encoder type

Reply:
my idea idea for kornia_rs was to provide very low level and explicit operators so that is easier later to be implemented in any python library (in the direction of low level dll ops libraries). Also i believe it's easier from kornia python side to have high level `read_image` or `write_image`  and we dispatch based on filenames or similar

Reply:
fair